### PR TITLE
[EDU-3160] feat: automate sec - edge functions and apis

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue
- [EDU-3160](https://aziontech.atlassian.net/browse/EDU-3160 )

Changes
- Traduz docs de automate secure - edge functions e APIs

[EDU-3160]: https://aziontech.atlassian.net/browse/EDU-3160?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+meta_tags: edge, security, javascript
+namespace: documentation_secure_automate_functions
+menu_namespace: secureMenu
+permalink: /documentacao/produtos/secure/automate/functions/
+---
+
+import Button from '~/components/Button.astro'
+
+
+## Edge Functions e Edge Firewall
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx:13]: pensando se é necessário esse h2 aqui

File: src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Edge functions são funções executadas na plataforma de edge da Azion, com baixa latência, trazendo o poder operacional mais perto do usuário final.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx:15]:
```suggestion
Edge functions são funções executadas na Plataforma de Edge da Azion, com baixa latência, trazendo o poder operacional mais perto do usuário final.
```

File: src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+<Button href="/pt-br/documentacao/produtos/devtools/azion-edge-runtime/compatibilidade/node/" text="saiba mais sobre a compatibilidade com node.js" variant="secondary" target="_blank"></Button>
+
+### Metadados
+
+A API de Metadados permite o acesso a um conjunto de metadados que podem ser manipulados para:
+
+- Filtrar e gerenciar o acesso à sua aplicação
+- Aplicar lógica específica em diferentes cenários com base no GeoIP
+
+<Button href="/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/api-reference/metadata/" text="ir para referência da API de metadados" variant="secondary" target="_blank"></Button>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx:57]:
```suggestion
<Button href="/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/api-reference/metadata/" text="saiba mais sobre a API de metadados" variant="secondary" target="_blank"></Button>
```

File: src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+### Network list
+
+A Interface de network list da Azion permite comparar um endereço IP com uma lista de rede específica. Se o endereço IP informado estiver dentro da lista de rede, ele retorna `true` e a lógica de implementação desejada pode continuar com base nessa informação.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx:61]:
```suggestion
A interface de network list da Azion permite comparar um endereço IP com uma lista de rede específica. Se o endereço IP informado estiver dentro da lista de rede, ele retorna `true` e a lógica de implementação desejada pode continuar com base nessa informação.
```

File: src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+Os IPs listados podem ser usados para gerenciamento de:
+
+- Allowlists
+- Blocklists
+- Greylists
+
+<Button href="/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/api-reference/network-list/" text="ir para referência da API de network list" variant="secondary" target="_blank"></Button>
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx:69]:
```suggestion
<Button href="/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/api-reference/network-list/" text="saiba mais sobre a API de network list" variant="secondary" target="_blank"></Button>
```

File: src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+---
+
+## Desenvolva edge functions para a segurança
+
+As edge functions da Azion para o firewall são executadas na fase de requisição e seguem o seguinte processo:
+
+1. As regras configuradas no [Rules Engine para Edge Firewall](/pt-br/documentacao/produtos/edge-firewall/rules-engine/) para a função ser executada são acionadas.
+
+2. O Azion Edge Runtime processa a função, retornando um resultado.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx:33]:
```suggestion
2. O Edge Runtime da Azion processa a função, retornando um resultado.
```

File: src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+3. O Rules Engine para Edge Firewall retoma o processamento, com base no resultado, a partir do ponto em que o comportamento foi acionado.
+
+
+<Button href="/pt-br/documentacao/produtos/edge-firewall/edge-functions/firewall/" text="saiba mais sobre edge functions para edge firewall" variant="secondary" target="_blank"></Button>
+
+---
+
+## Segurança através de APIs
+
+[Azion Edge Runtime](/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/visao-geral/) suporta uma lista de APIs que permitem a manipulação de [metadados](/pt-br/documentacao/produtos/secure/automate/functions/#metadados) e o uso de [network lists](/pt-br/documentacao/produtos/secure/automate/functions/#network-list) para ajudar na implementação da lógica.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx:44]:
```suggestion
O [Edge Runtime da Azion](/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/visao-geral/) suporta uma lista de APIs que permitem a manipulação de [metadados](/pt-br/documentacao/produtos/secure/automate/functions/#metadados) e o uso de [network lists](/pt-br/documentacao/produtos/secure/automate/functions/#network-list) para ajudar na implementação da lógica.
```

**Review by MarianaAguilera — APPROVED** [review]
algumas sugestões menores

File: src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx [hunk 8]
```diff
@@ -0,0 +1,76 @@
+---
+title: Automatize a segurança com Azion Edge Functions
+description: Através das edge functions, você pode escrever seu próprio código de segurança em JavaScript e implantá-lo para ser executado no edge.
+meta_tags: edge, security, javascript
+namespace: documentation_secure_automate_functions
+menu_namespace: secureMenu
+permalink: /documentacao/produtos/secure/automate/functions/
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx:7]:
```suggestion
permalink: /documentacao/produtos/secure/automatize/functions/
```

**Review by bru-andrade — APPROVED** [review]
Asmei

File: src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx (hunk 7 above)
Comment (gabriel-azion) [reply src/content/docs/pt-br/pages/secure-jornada/automate/edge-functions-apis.mdx:44]:
```suggestion
O [Azion Edge Runtime](/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/visao-geral/) suporta uma lista de APIs que permitem a manipulação de [metadados](/pt-br/documentacao/produtos/secure/automate/functions/#metadados) e o uso de [network lists](/pt-br/documentacao/produtos/secure/automate/functions/#network-list) para ajudar na implementação da lógica.
```

**Pull request merged**